cypress_test/integration_tests/common/helper.js

```diff
@@ -348,7 +348,7 @@ function waitForInterferingUser() {
 function documentChecks(skipInitializedCheck = false) {
 	cy.log('>> documentChecks - start');
 
-	cy.cGet('#document-canvas', {timeout : Cypress.config('defaultCommandTimeout') * 2.0});
```

Comment:
Allowing document to load 300s seems to be too much (default is 10s * 2 IIRC).

CI is in a good shape now so not a problem. 
When we can't get canvas there is most often problem on the server side, some error which kills websocket or similar. Did you try to run this locally and debug? `make run-desktop` in cypress_test directory. See logs in the browser, requests. Also there are error.log in workdir which might helpful to see.

Reply:
I was testing the values, but it still fails. I tried running it locally with run-mobile spec=something, and it loads the canvas and starts testing the unit tests. However, these changes should not affect loading the JavaScript bundle or initializing the map. Interestingly, I will start adding logs to identify the issue.